File: 2455.c
#include <stdio.h>

int main() {

	int p1, c1, p2, c2;

	scanf("%d %d %d %d", &p1, &c1, &p2, &c2);

	int pc1, pc2;

	pc1 = p1*c1;
	pc2 = p2*c2;

	if (pc1 == pc2) {
		printf("%d\n", 0);

	} else if (pc1 > pc2) {
		printf("%d\n", -1);

	} else {
		printf("%d\n", 1);
	}

	return 0;
}

File: 2455.py
p1, c1, p2, c2 = map(int, input().split())

pc1 = p1*c1
pc2 = p2*c2

if pc1 == pc2:
	print(0)

elif pc1 > pc2:
	print(-1)

else:
	print(1)